network/src/commonMain/kotlin/com/wire/kalium/network/api/user/login/SSOLoginApi.kt

```diff
@@ -57,6 +66,31 @@ class SSOLoginApiImpl(private val httpClient: HttpClient) : SSOLoginApi {
         }
     }
 
+    override suspend fun ssoEstablishSession(cookie: String, apiBaseUrl: Url): NetworkResponse<SessionDTO> = wrapKaliumResponse<AccessTokenDTO> {
+            httpClient.post {
+                setUrl(apiBaseUrl, PATH_ACCESS)
```

Comment:
I'm thinking if it should be in the `SSOLoginApi` as it's not a request dedicated exclusively to the SSO 🤔

Reply:
I get your point, but we already such a function with different logic in the login api and this one is specifically for the sso login.